Imperia 1.7 final fixes

Unintuition-Time opened the pull request:
90% done, just needs some possible error fixes for china stuff, poland, and true neutral

Unintuition-Time committed:
Power Bloc & Qing Fixes

Unintuition-Time committed:
State Trait Russia & Tanzimat Fixes

Unintuition-Time committed:
Readded German Minors to Bloc

& Decree and Tanzimat change

Unintuition-Time committed:
Defines Tweaks

Unintuition-Time committed:
Deprecation & Diplomatic Pact Cost

Unintuition-Time committed:
Moved Tanzimat to replace; Localization

will see if this fixes errors

Unintuition-Time committed:
Error Smashing

Plus a couple minor changes

Danarca committed:
Neutrality & Sheepicons added

Danarca committed:
Re-encoded txt files to UTF-8 BOM

Danarca commented on common/history/power_blocs/00_power_blocs.txt:
Are we sure we want these German minors to be in the British empire, if we can't remove them again?

HiddeLekanne replied:
Why do we need to add them in the first place? aren't they subjects?

Danarca replied:
They are independent in this version of the game.
Used to be that they would leave the British market with Hannover, but right now they are independent unless/until Prussia invites them to their Sphere.

Unintuition-Time committed:
Subjects

Unintuition-Time committed:
Merge branch 'Imperia-1.7-Final-Fixes' of https://github.com/Imperia-Development-Victoria-3/Victoria-3-Dev into Imperia-1.7-Final-Fixes

Unintuition-Time committed:
Hannover Fix

Unintuition-Time committed:
ACW JE

Unintuition-Time committed:
ACW Change

Removes primary culture for Afro-Americans and replaces it with a modifier

Unintuition-Time committed:
Images & Minor Tweaks

American colonization rates, grenade change

Unintuition-Time committed:
Added Mines PM

Unintuition-Time committed:
Mortality Rate Changes

For these pms, an outsized amount of deaths would occur from this mortality bonus. This will mitigate that.

Unintuition-Time committed:
Merge branch 'Imperia-Core' of https://github.com/Imperia-Development-Victoria-3/Victoria-3-Dev into Imperia-1.7-Final-Fixes

Unintuition-Time committed:
Merge Conflict Fix
